Sasha/Files/zip_maker/zip_maker.py

```diff
@@ -0,0 +1,45 @@
+import zipfile
+import os
+import sys
+
+def get_cmd_files(arg_list):
+    files = []
+    
+    for filename in arg_list[1:]: # start from second item in command line
+        files.append(filename)
+
+    return files    
+
+def get_all_files():
+    current_dir = os.getcwd()
+    files = []
+
+    for filename in os.listdir(current_dir):
+        if filename.startswith("."): # skip hidden files and folders
+            continue
+        
+        files.append(filename)
+
+    return files
+
+def zip_all_in_dir(files):
+    with zipfile.ZipFile('arch.zip', 'w', compression=zipfile.ZIP_DEFLATED) as zip_arch: 
+        for fname in files:
+            zip_arch.write(fname)
+        
+        zip_arch.close()
+
+def main():
+    input_files = sys.argv
+    
+    if len(input_files) >= 2:
+        files = get_cmd_files(input_files)
```

Comment:
Why are you checking length? As I got you have one input for folder and more for files.. As I see you are using command line parameters. Please do it with '-d' for folder and '-f' for list of files